glslc/src/main.cc

```diff
@@ -133,8 +138,12 @@ int main(int argc, char** argv) {
       compiler.SetWorkingDirectory(workdir.str());
     } else if (arg.starts_with("-fshader-stage=")) {
       const string_piece stage = arg.substr(std::strlen("-fshader-stage="));
-      force_shader_stage = glslc::GetShaderStageFromCmdLine(arg);
-      if (force_shader_stage == EShLangCount) {
+      // forced_shader_kind can be 1) one of the forced shader kinds, which
+      // means following files will be compiler with the specified shader
+      // kind; 2) shaderc_glsl_infer_from_source, which means 'arg' doesn't
```

Comment:
This is a weird result to get in case of an invalid arg.  Why not explicitly indicate an error?